Nguyen_Ken/Assignments/multiples.py

```diff
@@ -0,0 +1,10 @@
+#multiples assignment part 1
+for count in range(1,1001):
+    print count
+    count+=2
```

Comment:
Technically this works (in python), but it's incorrect. the `for` statement iterates over the range. even though you adjust the count within the loop, that adjustment is _completely_ ignored. So if you run this for-loop, it'll actually print 1,2,3,4,5,etc rather than up by two.

There are two ways you could get to the right answer. One method would be to iterate up from 1,1001 by 1 (as you're doing above) but adding a conditional into the loop so that you only display even #s. Another technique would be to dig deeper into the documentation for python's [range function](http://pythoncentral.io/pythons-range-function-explained/) and see if there's anything in there that'll help you out 😇.